### PR TITLE
622 exportinput

### LINKED ISSUE #622: export input does not work as intended all inputs should be exported and includemetavars array so like country in the array

### PR DESCRIPTION
**Pull request by Incin**

### CODE REVIEW

File: src/components/DownloadCsv/DownloadCsv.tsx [hunk 1]
```diff
@@ -1,37 +1,117 @@
...
+    const csvString = [cols.join(','), ...rows.map((d) => d.join(','))].join('\n');
+    const csvFile = new Blob([csvString], { type: 'text/csv' });
+    const downloadLink = document.createElement('a');
+    downloadLink.download = `${displayInfo.name}_${fullName}_${email}_${jobTitle}_${
+      new Date().toISOString().split('T')[0]
+    }.csv`;
+    downloadLink.href = window.URL.createObjectURL(csvFile);
+    downloadLink.click();
+    setCsvDownloaded(true);
+  };
```
Comment (jefferymills) [inline src/components/DownloadCsv/DownloadCsv.tsx:90]: We should make a future task to extract all this logic out of the component.

File: src/components/UserInfo/UserInfo.tsx [hunk 2]
```diff
@@ -37,53 +23,89 @@ const UserInfo: React.FC<UserInfoProps> = ({
...
+
+  const handleJobTitleChange = (event: { target: { value: string } }) => {
+    setJobTitle(event.target.value);
+    localStorage.setItem(storageItems[2], event.target.value);
+  };
+
+  const handleOtherInfoChange = (event: { target: { value: string } }) => {
+    setOtherInfo(event.target.value);
+    localStorage.setItem(storageItems[3], event.target.value);
+  };
```
Comment (jefferymills) [inline src/components/UserInfo/UserInfo.tsx:63]: I also wonder if in a future task could be to consolidate all the individual save methods and state properties.

**Review by jefferymills — APPROVED** [review]
🗝️

File: src/components/DownloadCsv/DownloadCsv.tsx (hunk 1 above)
Comment (Incin) [reply src/components/DownloadCsv/DownloadCsv.tsx:90]: tickets have been made for all of these suggestions

File: src/components/UserInfo/UserInfo.tsx (hunk 2 above)
Comment (Incin) [reply src/components/UserInfo/UserInfo.tsx:63]: tickets have been made for all of these suggestions

**Pull request merged**